internal/extensionapi/constants.go

```diff
@@ -0,0 +1,6 @@
+package extensionapi
+
+const (
+	// VSCodeScheme is the URL scheme for VSCode remote connections
+	VSCodeScheme = "vscode://amazonwebservices.aws-toolkit-vscode/connect/sagemaker"
```

Comment:
fast-follow: move any reference to `/sagemaker`, `aws` etc to the `aws` package.